<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>top</class>
 <widget class="QWidget" name="top">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>880</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	border-image: url(:/imager/2000.jpg);
}
QPushButton#but1
{
	border-image: url(:/imager/2201.png);
}
QPushButton#but2
{
	
	border-image: url(:/imager/2002.png);
}
QPushButton#but3
{
	
border-image: url(:/imager/2003.png);
}
QPushButton#but4
{
	
	border-image: url(:/imager/2005.png);
}
QPushButton#but5
{
	border-image: url(:/imager/2007.png);
}
QPushButton#but6
{
	
	border-image: url(:/imager/2006.png);
}
QPushButton#but7
{
  
	border-image: url(:/imager/2004.png);
}
.QTextEdit
{
  background-color:rgb(35,28,46);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>580</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>880</width>
     <height>580</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>880</width>
     <height>580</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but1">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>190</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#fce94f;&quot;&gt;消防安全疏散管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imager/2024.png);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
